src/main/java/org/elasticsearch/index/mapper/geo/GeoPointFieldMapper.java

```diff
@@ -254,25 +250,27 @@ public GeoPointFieldMapper build(BuilderContext context) {
                     }
                     iterator.remove();
                 } else if (fieldName.equals("validate")) {
-                    builder.validateLat = XContentMapValues.nodeBooleanValue(fieldNode);
-                    builder.validateLon = XContentMapValues.nodeBooleanValue(fieldNode);
+                    builder.validate = XContentMapValues.nodeBooleanValue(fieldNode);
                     iterator.remove();
                 } else if (fieldName.equals("validate_lon")) {
-                    builder.validateLon = XContentMapValues.nodeBooleanValue(fieldNode);
-                    iterator.remove();
+                    if (parserContext.indexVersionCreated().onOrAfter(Version.V_1_6_0)) {
```

Comment:
I like that idea. I'm a big fan of avoiding exceptions if at all possible but wasn't sure how we've handled bwc in other cases. That will let us remove the 'breaking' label too?  Committing a change shortly for review.  Will also need to add more thorough testing before merging. I just needed this valuable feedback.